src/Adapter/Product/Combination/CommandHandler/UpdateCombinationStockHandler.php

```diff
@@ -28,37 +28,27 @@
 
 namespace PrestaShop\PrestaShop\Adapter\Product\Combination\CommandHandler;
 
-use Combination;
-use PrestaShop\PrestaShop\Adapter\Product\Combination\Repository\CombinationRepository;
+use PrestaShop\PrestaShop\Adapter\Product\Combination\Update\CombinationStockProperties;
 use PrestaShop\PrestaShop\Adapter\Product\Combination\Update\CombinationStockUpdater;
 use PrestaShop\PrestaShop\Core\Domain\Product\Combination\Command\UpdateCombinationStockCommand;
 use PrestaShop\PrestaShop\Core\Domain\Product\Combination\CommandHandler\UpdateCombinationStockHandlerInterface;
-use PrestaShop\PrestaShop\Core\Util\DateTime\DateTime;
 
 /**
  * Handles @see UpdateCombinationStockCommand using legacy object model
  */
 final class UpdateCombinationStockHandler implements UpdateCombinationStockHandlerInterface
```

Comment:
Is there a behat test exist for this Handler ?

Reply:
tests/Integration/Behaviour/Features/Scenario/Product/Combination/update_combination_stock.feature